reactLab/src/Modal.js

```diff
@@ -0,0 +1,20 @@
+import './Modal.css';
+
+function Modal(props){
+    return (
+        <div className="modal">
+            <div className="modal-content">
+                {props.children}
+                <div id="modal-response">
+                    <button type="button" onClick={props.cancelOnClick}>
```

Comment:
You can have separate props.cancelOnClick (although that naming seems like it should be onCancelClick) and confirmOnClick.  However, as in the solution for https://github.com/nrhodes-hmc/cs124-fa21-react-alert-tab/blob/alert-solution/src/AlertApp.js, I think having onClose and onOK make more sense.  The onClose will do whatever common stuff will need to happen on a close, and the onOK will do whatever is additionally needed on a confirmation.